app/views/listings/index.html.haml

```diff
@@ -0,0 +1,9 @@
+%h1 Listings#index
+- @listings.each do |listing|
+  #listing
```

Comment:
What is the purpose of this div and how is the current indentation affecting what is being rendered by the browser?

Reply:
we will assign an id to every div (the id of the actual listing) in order to ensure and test that we have right details in the right listing (what Oliver is commenting below). This is work in progress, we have some trouble with Capibara not finding css but Oliver is looking at that.